tests/src/test/scala/org/scalanative/bindgen/BindgenSpec.scala

```diff
@@ -38,11 +48,35 @@ class BindgenSpec extends FunSpec {
         val expected = new File(inputDirectory, testName + ".scala")
         val output   = new File(outputDir, testName + ".scala")
 
-        bindgen(input, testName, output)
+        bindgen(input, testName, Option(output))
 
         assert(output.exists())
         assert(contentOf(output) == contentOf(expected))
       }
+
+      it(s"should print correct warnings for ${input.getName}") {
+        val testName = input.getName.replace(".h", "")
+
+        val errs = bindgen(input, testName)
+
+        testName match {
+          case "LiteralDefine" =>
+            assert(
+              errs == "Warning: integer value does not fit into 8 bytes: 18446744073709551615\n" +
+                "Warning: integer value does not fit into 8 bytes: 9223372036854775809")
```

Comment:
Sorry for not being more clear, what I meant by _"have checks for errors in a separate Spec"_ was to do something like:

```scala
class BindgenReportingSpec extends FunSpec {
  it("Skip variable with opaque type") {
    val result = bindgen(
      input = "struct undefinedStruct;
              |extern struct undefinedStruct removedExtern;
              |#define removedExternAlias removedExtern
              |".stripMargin)
    assert(
      result.error == "Error: Variable alias removedExternAlias is skipped because it has incomplete type")
    )
  }
}
```
That is avoid mixing code with errors into the header files used by `BindgenSpec` and instead have focused tests.

I know it is a bit more work but I think the clarity it brings will pay off in the long run.